### PR TITLE
New Fabric connector docs

### PR DESCRIPTION
**Pull request by davidkel**
available at https://davidkel.github.io/caliper/vNext/getting-started/

Signed-off-by: D <d_kelsey@uk.ibm.com>

### CODE REVIEW

File: docs/vNext/Fabric_Configuration_New.md [hunk 1]
```diff
@@ -0,0 +1,1115 @@
...
+> ```yaml
+> caliper:
+>   fabric:
+>     gateway:
+>       enabled: true
+> ```
+> After naming the [project settings](./Runtime_Configuration.md#project-level) file `caliper.yaml` and placing it in the root of your workspace directory, it will override the following two setting keys with the following values:
+> * Setting `caliper-fabric-gateway-enabled` is set to `true`
+>
+> __The other settings remain unchanged.__
```
Comment (nklincoln) [inline docs/vNext/Fabric_Configuration_New.md:97]: Only shows setting one item, but claims two. Might be a good opportunity to show some other common gateway settings such as `localhost: false`

File: docs/vNext/Fabric_Configuration_New.md [hunk 2]
```diff
@@ -0,0 +1,1115 @@
...
+
+### Binding with Fabric SDK 2.x
+> Note that when using the binding target for the Fabric SDK 2.x there are capability restrictions:
+> * The 2.x SDK does not facilitate administration actions. It it not possible to create/join channels, nor install/instantiate contract. Consequently the 2.x binding only facilitates operation with a `--caliper-flow-only-test` flag
+> * The 2.x SDK only supports operation using a `gateway`. Consequently the 2.x binding requires a `--caliper-fabric-gateway-enabled` flag
+
+### Using the Operational capabilities of the Adapters
+The ability to create channels and perform install/instantiate of contracts (chaincode) remains in the new 1.x sdk based adapters, and nothing equivalent exists for the 2.x sdk based adapters. This capability remains mainly to support the caliper integration tests and the use of these mechanisms to setup your channels and contracts is highly discouraged. This capability is considered legacy and will not be maintained. You should look into alternative methods for setting up your network prior to running caliper to benchmark it.
```
Comment (nklincoln) [inline docs/vNext/Fabric_Configuration_New.md:77]: Upper case for Caliper, probably worth a global search/replace (won't mention other occurrences in further review comments)

File: docs/vNext/Fabric_Configuration_New.md [hunk 3]
```diff
@@ -0,0 +1,1115 @@
...
+user@ubuntu:~/caliper-benchmarks$ export CALIPER_FABRIC_SKIPCREATECHANNEL_MYCHANNEL=true
+```
+
+> __Note:__ This settings is intended for easily skipping the creation of a channel that is specified in the network configuration file as "not created". However, if you know that the channel always will be created during benchmarking, then it is recommended to denote this explicitly in the network configuration file.
+
+Naturally, you can specify the above setting multiple ways (e.g., command line argument, configuration file entry).
+
+## The connector API
+
+The [workload modules](./Workload_Module.md) interact with the adapter at two phases of the tests: during the initialization of the user module (in the `initializeWorkloadModule` callback), and when submitting invoke or query transactions (in the `submitTransaction` callback).
```
Comment (nklincoln) [inline docs/vNext/Fabric_Configuration_New.md:113]: there are 3 phases - there is an optional cleanup stage too

**Review by nklincoln — CHANGES_REQUESTED** [review]
A few minor things, but looking great!

File: docs/vNext/Fabric_Configuration_New.md (hunk 2 above)
Comment (davidkel) [reply docs/vNext/Fabric_Configuration_New.md:77]: Only found a couple to be done which were in this section.

File: docs/vNext/Fabric_Configuration_New.md (hunk 1 above)
Comment (davidkel) [reply docs/vNext/Fabric_Configuration_New.md:97]: Definitely added in localhost.

File: docs/vNext/Fabric_Configuration_New.md (hunk 3 above)
Comment (davidkel) [reply docs/vNext/Fabric_Configuration_New.md]: Thanks Nick, added a bit about cleanup

File: docs/vNext/Fabric_Configuration_New.md [hunk 4]
```diff
@@ -0,0 +1,1122 @@
...
+user@ubuntu:~/caliper-benchmarks$ export CALIPER_FABRIC_SKIPCREATECHANNEL_MYCHANNEL=true
+```
+
+> __Note:__ This settings is intended for easily skipping the creation of a channel that is specified in the network configuration file as "not created". However, if you know that the channel always will be created during benchmarking, then it is recommended to denote this explicitly in the network configuration file.
+
+Naturally, you can specify the above setting multiple ways (e.g., command line argument, configuration file entry).
+
+## The connector API
+
+The [workload modules](./Workload_Module.md) interact with the adapter at three phases of the tests: during the initialization of the user module (in the `initializeWorkloadModule` callback), when submitting invoke or query transactions (in the `submitTransaction` callback), and at the optional cleanup od the user module (in the `cleanupWorkloadModule` callback).
```
Comment (nklincoln) [inline docs/vNext/Fabric_Configuration_New.md:117]: od -> of typo

**Review by nklincoln — CHANGES_REQUESTED** [review]
small typo to fix

File: docs/vNext/Fabric_Configuration_New.md (hunk 4 above)
Comment (davidkel) [reply docs/vNext/Fabric_Configuration_New.md:117]: Thanks Nick, have fixed.

**Review by nklincoln — APPROVED** [review]
📖  🐛

**Pull request merged**